bin/citrea/tests/bitcoin_e2e/tests/prover_test.rs

```diff
@@ -63,9 +68,32 @@ impl TestCase for BasicProverTest {
         da.generate(5, None).await?;
         let finalized_height = da.get_finalized_height().await?;
         prover
-            .wait_for_l1_height(finalized_height, Some(Duration::from_secs(600)))
+            .wait_for_l1_height(finalized_height, Some(Duration::from_secs(300)))
             .await;
 
+        da.generate(5, None).await?;
+        let proofs = full_node
+            .wait_for_zkproofs(finalized_height + 5, Some(Duration::from_secs(120)))
+            .await
+            .unwrap();
+
+        {
+            // print some debug info about state diff
+            let state_diff = &proofs[0].state_transition.state_diff;
+            let state_diff_size: usize = state_diff
+                .iter()
+                .map(|(k, v)| k.len() + v.as_ref().map(|v| v.len()).unwrap_or_default())
+                .sum();
+            let borshed_state_diff = borsh::to_vec(state_diff).unwrap();
+            let compressed_state_diff =
+                bitcoin_da::helpers::compression::compress_blob(&borshed_state_diff);
+            println!(
+                "StateDiff: size {}, compressed {}",
+                state_diff_size,
+                compressed_state_diff.len()
+            );
+        }
```

Comment:
We may add another CI job. This test is not very useful because it proves only systxs (there are no other txs in blocks). So this debug print is more for local development to automate some things. But yeah, a big proof with different txs in CI would be interesting thing to have.